src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1423,6 +1459,28 @@ pre.rust {
 	animation: rotating 2s linear infinite;
 }
 
+.setting-line .radio-line input {
+	border-color: var(--main-background-color);
+}
```

Comment:
This looks wrong. Why would a border color be the same as the background color? That would make the border invisible. And in fact, looking in DevTools, this property seems to get overridden by the `border: 1px solid` rule in settings.css.

Can we remove this rule?

Reply:
It seems to be working the same without it so I remove it.